app/src/main/kotlin/com/hedvig/app/feature/insurance/ui/tab/InsuranceFragment.kt

```diff
@@ -73,6 +74,10 @@ class InsuranceFragment : Fragment(R.layout.fragment_insurance) {
             ?.navigate(requireContext())
             ?.also { insuranceViewModel.crossSellActionOpened() }
 
+          viewState.navigateWeb
+            ?.let { activity?.openWebBrowser(it) }
+            ?.also { insuranceViewModel.crossSellActionOpened() }
```

Comment:
Inside crossSellActionOpened() we'd need to then also clear the `navigateWeb` variable so that it doesn't stay true right? Otherwise if going back here the navigateWeb would still be there, and this would retrigger. That's what I gather from the code at least.

Reply:
Good catch, fixing 🙏